pycbc/conversions.py

```diff
@@ -550,6 +550,21 @@ def spin2y_from_mass1_mass2_xi2_phi_a_phi_s(mass1, mass2, xi2, phi_a, phi_s):
     chi_perp = chi_perp_from_mass1_mass2_xi2(mass1, mass2, xi2)
     phi2 = phi2_from_phi_a_phi_s(phi_a, phi_s)
     return chi_perp * numpy.sin(phi2)
+    
+def dquad_from_lambda(lambdav):
```

Comment:
If we're going to follow the lal convention, let's stick to it: `dquadmon_from_lambda`

Reply:
Ok, no reason to deviate.